src/contracts/digital-twin/container.ts

```diff
@@ -415,6 +420,29 @@ export class Container extends Logger {
     return (await profile.getPlugins() || { });
   }
 
+  /**
+   * Removes properties from a description that aren't necessary for a plugin / template export.
+   *
+   * @param      {any}          description        dbcp container description
+   * @param      {Arraystring}  allowedProperties  array of properties that should be kept
+   * @return     {any}          purged dbcp description
+   */
+  public static purgeDescriptionForTemplate(
+    description: any,
+    allowedProperties: Array<string> = ['author', 'dapp', 'dbcpVersion', 'description', 'i18n',
+      'imgSquare', 'imgWide', 'license', 'name', 'tags'],
+  ): any {
+    const purged: any = { };
+    allowedProperties.forEach((prop: string) => {
+      // eslint-disable-next-line no-prototype-builtins
+      if (description.hasOwnProperty(prop)) {
```

Comment:
would be more secure using `Object.prototype.hasOwnProperty.call(description, prop)` instead of disabling eslint rule

Reply:
see 
> Additionally, objects can have properties that shadow the builtins on Object.prototype, potentially causing unintended behavior or denial-of-service security vulnerabilities. For example, it would be unsafe for a webserver to parse JSON input from a client and call hasOwnProperty directly on the resulting object, because a malicious client could send a JSON value like {"hasOwnProperty": 1} and cause the server to crash.

Reply:
or even better maybe revers the logic to something like this?

`Object.keys(description).filter( key => allowedProperties.includes(key); )`